SPECS/kexec-tools/makedumpfile-printk-fix.patch

```diff
@@ -0,0 +1,608 @@
+This patch is a combination of the following commits:
```

Comment:
Same comment as crash-printk-fix.patch

Reply:
Note - This patch was generated the same way as the other